Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalTBModuleX.cc

```diff
@@ -209,17 +207,16 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
 #endif
   const auto& materials = args.value<std::vector<std::string> >("MaterialNames");  // Material names in each layer
   const auto& names = args.value<std::vector<std::string> >("VolumeNames");        // Names of each layer
-  const auto& layerThick = args.value<std::vector<double> >("Thickness");          // Thickness of the material
+  auto layerThick = args.value<std::vector<double> >("Thickness");                 // Thickness of the material
   std::vector<int> copyNumber;                                                     // Copy numbers (initiated to 1)
   copyNumber.resize(materials.size(), 1);
 #ifdef EDM_ML_DEBUG
   edm::LogVerbatim("HGCalGeom") << "DDHGCalTBModuleX: " << materials.size() << " types of volumes";
   for (unsigned int i = 0; i < names.size(); ++i)
-    edm::LogVerbatim("HGCalGeom") << "Volume [" << i << "] " << names[i] << " of thickness "
-                                  << convertCmToMm(layerThick[i]) << " filled with " << materials[i]
-                                  << " first copy number " << copyNumber[i];
+    edm::LogVerbatim("HGCalGeom") << "Volume [" << i << "] " << names[i] << " of thickness " << layerThick[i]
+                                  << " filled with " << materials[i] << " first copy number " << copyNumber[i];
 #endif
-  const auto& blockThick = args.value<std::vector<double> >("BlockThick");   // Thickness of each section
+  auto blockThick = args.value<std::vector<double> >("BlockThick");          // Thickness of each section
```

Comment:
why is this now a copy rather than const ref? It doesn't seem like the values are changed anywhere later.

Reply:
put back